nodejs/week3/meal-app/src/backend/api/meals.js

```diff
@@ -0,0 +1,169 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const pool = require("./../database");
+
+
+router.get("/", (req, res) => {
+    const maxPrice = req.query.maxPrice;
+    const availableReservations = req.query.availableReservations;
+    const title = req.query.title;
+    const createdAfter = req.query.createdAfter;
+    const limit = req.query.limit;
+
+    if (Object.entries(req.query).length === 0) {
+        pool.query("select * from meal", function(error, results, fields) {
+            if (error) {
+                console.log(error);
+            }
+            res.send(results);
+            // http://localhost:5000/api/meals
+        })
+    } else if (maxPrice) {
+        pool.query(`select * from meal where price < ${parseFloat(maxPrice)}`, function(error, results, fields) {
```

Comment:
You should use placeholder values, otherwise there is a big risk of SQL injections:

```suggestion
        pool.query(`select * from meal where price < ?`, parseFloat(maxPrice) function(error, results, fields) {
```